dev/wlp-gradle/subprojects/fat.gradle

```diff
@@ -214,6 +214,14 @@ task autoFVT {
     bndProps.setProperty('fat.minimum.java.level', minJavaLevel)
     bndProps.setProperty('micro.version', bnd('libertyBundleMicroVersion'))
     
+    // Add in database rotation properties, if they are present
```

Comment:
I don't believe this is needed, any properties that start with `fat.*` or `test.*` should get added already up on line 209.
If this is needed because `System.getProperties()` has things that `bndWorkspace.getProject(project.getProjectDir()).getProperties()` doesn't, then we should add a generic mechanism for `System` properties in the same way we do for bnd project properties